### PR TITLE
John/bloglist

### PR DESCRIPTION
**Pull request by Mac-Genius**
The 'Continue Reading' button now expands the blog post instead of taking the user to another page.

### CODE REVIEW

File: auacm/angular/src/app/blog-list/blog-list.component.html [hunk 1]
```diff
@@ -18,9 +18,13 @@
           </h4>
         </div>
       </div>
-      <div class="text-left" [innerHTML]="converter.makeHtml(post.body | limitWords : 200)"></div>
-      <a *ngIf="post.resized" class="btn btn-default pull-right"
-          [routerLink]="['/blog/' + post.id]" role="button">Continue Reading</a>
+      <div class="text-left" [innerHTML]="converter.makeHtml(post.displayToggle ? post.body : post.body | limitWords : 200)"></div>
+      <!--<a *ngIf="post.resized" class="btn btn-default pull-right"
```
Comment (mcprice30) [inline auacm/angular/src/app/blog-list/blog-list.component.html]: Can we remove this comment?

File: auacm/angular/src/app/models/blog.ts [hunk 2]
```diff
@@ -6,13 +6,15 @@ export class BlogPost {
   public body: string;
   public author: BlogAuthor;
   public resized: boolean;
+  public displayToggle: boolean;
```
Comment (mcprice30) [inline auacm/angular/src/app/models/blog.ts]: I feel like `displayToggle` isn't perhaps the best name... I just feel like saying `displayToggle` is `true` doesn't really tell you much information about how the blog post is being rendered. Perhaps `isExpanded` or something similar will be more descriptive?

**Review by mcprice30 — COMMENTED** [review]
Not necessary changes, just some readability stuff.

**Review by mcprice30 — APPROVED** [review]
LGTM

**Review by BrandonLMorris — APPROVED** [review]
lgtm

**Pull request merged**